projects/opentrain/gtfs_pipeline/data_handler/DataHandler.java

```diff
@@ -1,24 +1,560 @@
 package projects.opentrain.gtfs_pipeline.data_handler;
-    
+
+//imports for proto

```

Comment:
I think comments for imports aren't required. If you need it please use a single convention for the whole project. We are trying start comment with a capital letter. Also below.

E.g.

```
// It's a comment
```